source/game/field/obj/ObjFlow.hh

```diff
@@ -0,0 +1,76 @@
+#pragma once
+#include <Common.hh>
+
+namespace Field {
+
+class ObjAttrs {
+public:
+    u16 id() const {
```

Comment:
Missing ctor. Even if it's default, you should declare it to match convention. In the source, define as:
```c++
ObjAttrs::ObjAttrs() = default;
```

Reply:
This is a POD that needs access specifiers because it's accessing big-endian data from ObjFlow.bin in Common.szs. If not for that I would have made it a struct. I never actually construct an instance of this class, rather I reinterpret_cast an offset into the file as ObjAttrs*. I deleted default constructors though, as we talked about earlier.